foobarbaz/urls.py

```diff
@@ -1,12 +1,26 @@
 from django.conf import settings
-from django.conf.urls import patterns, url
+from django.conf.urls import patterns, url, include
+from django.contrib import admin
 #from django.views.generic import TemplateView
 from foobarbaz.views import views
 
 urlpatterns = patterns('',
     #url(r'^foo$', views.foo, name='foo'),
+    url(r'^admin/', include(admin.site.urls)),
     url(r'^login$', 'django.contrib.auth.views.login', {'template_name': 'login.html'}),
     url(r'^logout$', views.logout, name='logout'),
     url(r'^$', views.index, name='index'),
     url(r'^static/(?P<path>.*)$', 'django.views.static.serve', {'document_root': settings.STATIC_ROOT}),
+    # Password Reset URLs:
+    url(r'^accounts/password_reset/$', 
+        'django.contrib.auth.views.password_reset', 
+        {'post_reset_redirect' : '/accounts/password_reset/mailed/'},
+        name="password_reset"),
+    (r'^accounts/password_reset/mailed/$',
```

Comment:
Maybe we should look into pulling these into another auth_urls.py file to make it easier to merge?